bimmer_connected/cli.py

```diff
@@ -46,6 +46,11 @@ def main_parser() -> argparse.ArgumentParser:
     flash_parser.add_argument("vin", help=TEXT_VIN)
     flash_parser.set_defaults(func=light_flash)
 
+    flash_parser = subparsers.add_parser("horn", description="Flash the vehicle lights.")
+    _add_default_arguments(flash_parser)
+    flash_parser.add_argument("vin", help=TEXT_VIN)
+    flash_parser.set_defaults(func=horn)
```

Comment:
Please rename to `horn_parser`. Should work anyway, but is much clearer.

Reply:
@rikroe I adjusted the changes as wished. sorry, I oversaw the variable name.

Reply:
Changed